### PR TITLE
fix: #1753 added switches for secure boolean UI

### PR DESCRIPTION
**Pull request by CursedRock17**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This was a bug fix in response to issue #1753.

- **Why was this change needed?** (You can also link to an open issue here)
In that issue somebody brought up features that weren't being used according to the docs so they should be removed. I both removed the Sender name for nodemailer and added switches instead of input for all booleans, specifically the Secure field. 

- **Other information**:

I looked through the documentation and noticed many other documentation pages are either missing information, or the page is incorrect and needs to be fixed.

### CODE REVIEW

**Review by scopsy — APPROVED** [review]
Perfect! Could you just revert the image path?

File: docs/docs/platform/digest.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This becomes useful when a user needs to be notified on a large amount of trigge
 
 After adding a digest node in the workflow editor, each node that will be below the digest node will be only triggered once in the specified digest interval. You can decide to send messages before adding a digest node and they will be triggered in real-time.
 
-![Workflow Photo](/img/digest-flow.png)
+![Workflow Photo](../../static/img/digest-flow.png)
```
[the remark this answers is not in the document]
Comment (CursedRock17) [reply docs/docs/platform/digest.md]: That should have reverted the screenshot change, saw it working on docasaurus as well.

[the remark this answers is not in the document]
Comment (scopsy) [reply docs/docs/platform/digest.md]: Thanks!

**Comment by scopsy** [comment]
@CursedRock17 from the docs of nodemailer, seems like we can add the support to add the from name with the following syntax:

> from - The email address of the sender. All email addresses can be plain ‘[sender@server.com](mailto:sender@server.com)’ or formatted '“Sender Name” [sender@server.com](mailto:sender@server.com)', see [Address object](https://nodemailer.com/message/addresses/) for details

https://nodemailer.com/message/ 

Maybe we can create conditional the from string if a sendername exists? instead of removing this configuration option from the modal? 

What do you think?

**Comment by CursedRock17** [comment]
I definitely think that's a good idea, gives the users more options. Though, would this be better as an overall change, I haven't looked at the documentation for all of the providers in depth yet, but would this work for most of them as well. Just wondering if we should change the UI in general or if you just wanted a nodemailer provider fix here.

**Comment by scopsy** [comment]
@CursedRock17 seems like we have here some unrelated commits regarding pushwoosh. Do you mind separating them to a different PR? 

Regarding the addition of the sender name, I think this change is only missing in nodemaielr since other provider are familiar with the senderFrom property on the API level. I believe we can only fix nodemailer in this PR and no need for other changes. Wdyt?

**Comment by CursedRock17** [comment]
@scopsy I agree we most likely don't need to make any other changes. As for pushwoosh I had just made a template for the new provider and it wasn't letting me create a new pull request, I'll get that cleared.

**Comment by CursedRock17** [comment]
@scopsy I'm going to close this pull request since it has all been merged.

**Comment by CursedRock17** [comment]
@scopsy I know it's very messy, but I should've gotten rid of the pushwoosher provider API for this Pull Request. If you can merge and close this one I will push that provider into a new Pull Request

**Review by scopsy — APPROVED** [review]

**Review by p-fernandez — CHANGES_REQUESTED** [review]

File: libs/shared/src/consts/providers/credentials/provider-credentials.ts [hunk 2]
```diff
@@ -163,7 +163,7 @@ export const nodemailerConfig: IConfigCredentials[] = [
     type: 'string',
     required: false,
   },
-  ...mailConfigBase,
+  mailConfigBase[0],
```
[the remark this answers is not in the document]
Comment (CursedRock17) [reply libs/shared/src/consts/providers/credentials/provider-credentials.ts:166]: No it doesn't I was having merging issues with other branches I should have cleared it up and updated the branch.

File: README.md [hunk 3]
```diff
@@ -58,22 +58,21 @@ Building a notification system is complicated. At first, it seems like just send
...
 - [Embeddable notification center](https://github.com/novuhq/novu#embeddable-notification-center)
--  [Providers](https://github.com/novuhq/novu#providers)
-   - [Email](https://github.com/novuhq/novu#-email)
-   - [SMS](https://github.com/novuhq/novu#-sms)
-   - [Push](https://github.com/novuhq/novu#-push)
-   - [Chat](https://github.com/novuhq/novu#-chat)
-   - [In-App](https://github.com/novuhq/novu#-in-app)
-   - [Others](https://github.com/novuhq/novu#other-coming-soon)
-- [Need Help?](https://github.com/novuhq/novu#-need-help)
+- [Providers](https://github.com/novuhq/novu#providers)
```
[the remark this answers is not in the document]
Comment (CursedRock17) [reply README.md:65]: I apologize I don't know why that changed so slightly.

**Pull request closed without merging**